thirdParty/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if(BUILD_TESTING)
         add_compile_definitions($<$<CXX_COMPILER_ID:PGI>:__GCC_ATOMIC_TEST_AND_SET_TRUEVAL=1>)
         add_subdirectory(catch2)
     else()
-        find_package(Catch2 3.1.1 CONFIG REQUIRED)
+        find_package(Catch2 3.2.1 CONFIG REQUIRED)
```

Comment:
Just because we ship Catch 3.2.1 internally, does not mean that this is the minimum version we require? Can we require a lower version number here?

Reply:
That's a pity. I installed it globally to avoid rebuilding it, which used to happen several times a day. Anyway, I can reintall the newest one if necessary.